deploy/template.yaml

```diff
@@ -460,9 +460,10 @@ Resources:
       HealthCheckEnabled: TRUE
       HealthCheckProtocol: HTTP
       HealthCheckPath: /healthcheck
-      HealthCheckTimeoutSeconds: 14
-      HealthCheckIntervalSeconds: 15
+      HealthCheckTimeoutSeconds: 30
```

Comment:
Might be worth reviewing these values - my changes here need discussion, they're not recommendations.